packages/hardhat-core/src/internal/hardhat-network/provider/modules/debug.ts

```diff
@@ -4,13 +4,18 @@ import {
   RpcDebugTracingConfig,
 } from "../../../core/jsonrpc/types/input/debugTraceTransaction";
 import { validateParams } from "../../../core/jsonrpc/types/input/validation";
-import { MethodNotFoundError } from "../../../core/providers/errors";
+import {
+  InvalidArgumentsError,
+  MethodNotFoundError,
+} from "../../../core/providers/errors";
 import { HardhatNode } from "../node";
 import { RpcDebugTraceOutput } from "../output";
 
 /* eslint-disable @nomicfoundation/hardhat-internal-rules/only-hardhat-error */
 
 export class DebugModule {
+  public static readonly supportedTracers = ["structLogs"];
```

Comment:
I don't think there's a `structLogs` value for the tracer option in geth.

I will look into this more in depth tomorrow.